text/####-error-mitigation.md

```diff
@@ -0,0 +1,235 @@
+| **Updated**       | YYYY-MM-DD                                   |
+
+## Summary
+Error mitigation can greatly improve results on noisy quantum hardware.
+This is done by running a given quantum circuit several times.
+In run i of the circuit the duration of all the gates is stretched by a factor c_i to increase the noise in the gates.
+An improved result is then obtained by extrapolating to the zero-order noise limit c->0.
+The purpose of this RFC is to discuss how to implement error mitigation in Qiskit such that it is usable in applications.
+
+## Motivation
+Error mitigation allows users to significantly improve their results.
+It is expected that partners and members of the IBM Q network will benefit from this by being able to easily improve the quality of their results.
+
+## User Benefit
+The target users of this work are those who run quantum circuits for applications of quantum computing.
+
+## Design Proposal
+There are several ways to implement error mitigation.
+
+### Simple error mitigation
+The simplest way to implement error mitigation is to run the quantum circuit several times. 
+In each run the entangling two-qubit gate is replaced several entangling two-qubit gates such that the additional gates have no effect.
+For instance, replacing each CNOT gate in a quantum circuit by an odd number of CNOT gates results in the same quantum circuit in the ideal case.
+This has, for example, been implemented in https://arxiv.org/abs/1905.02666.
+This approach is also valid for other types of two-qubit gates such as the CZ gate.
+
+See https://github.com/Qiskit/qiskit-aqua/pull/683 which aims to implement this error mitigation method.
+
+**The advantage of this method is:**
+- Simplicity, it is easy to implement and understand.
+
+**The limitations of this method are**:
+- Quantum circuits that are already very deep may not see any gain since replacing each CNOT gate with three CNOT gates may produce circuits which, when executed, result in noise only.
+- It does not include single-qubit gates.
+- The effective stretch factors to chose from are very limited.
+- Some two-qubit gates, such as root-SWAP, need to be applied more than twice to compose to the identity.
+
+### Backend constrained error mitigation
+When implementing error mitigation using stretch factors, as is done in https://arxiv.org/abs/1805.04492, new pulses must be defined and calibrated for the different stretch factors c_i.
+To implement error mitigation in the manner the backend could have a set of pre-defined calibrated pulses with different stretch factors.
+For instance, following Kandala et al., the backend could store calibrated pulses for c=1 (i.e. the pulses used in regular operations), c=1.1, c=1.25, and c=1.5.
+At execute time, the user would specify that he wants to run a quantum circuit using error mitigation.
+The pulse scheduler would then create four copies of the quantum circuit, each with a different stretch factor supported by the backend.
```

Comment:
Further down you say this is the responsibility of the assembler. I think that's correct. The pulse scheduler should not be involved in any of this.
```suggestion
The assembler would then create four copies of the quantum circuit, each with a different stretch factor supported by the backend.
```

Reply:
Yes you are correct. Thanks.